lockbox-ios/Presenter/FxAPresenter.swift

```diff
@@ -22,6 +22,23 @@ class FxAPresenter {
     fileprivate let dispatcher: Dispatcher
     fileprivate let accountStore: AccountStore
 
+    fileprivate var _credentialProviderStore: Any?
+
+    @available (iOS 12, *)
+    private var credentialProviderStore: CredentialProviderStore? {
```

Comment:
It doesn't look like this will ever return nil, why is it optional?

Reply:
good catch!